docs/language/operators/from.md

```diff
@@ -54,20 +66,106 @@ from ... | switch color (
 ) | ...
 ```
 
-The output of a fork consists of multiple legs that must be merged.
-If the downstream operator expects a single input, then the output legs are
-merged with an automatically inserted [combine operator](combine.md).
+### Input Data
+
+Examples below below assume the existence of the Zed lake created and populated
+by the following commands:
+
+```mdtest-command
+export ZED_LAKE=example
+zed -q init
+zed -q create coinflips
+echo '{flip:1,result:"heads"} {flip:2,result:"tails"}' | zed -q -use coinflips load -
+zed -q -use coinflips branch trial 
+echo '{flip:3,result:"heads"}' | zed -q -use coinflips@trial load -
+zed -q create numbers
+echo '{number:1,word:"one"} {number:2,word:"two"} {number:3,word:"three"}' | zed -q -use numbers load -
+zed query -f text 'from :branches | yield pool.name + "@" + branch.name | sort'
+```
+
+The lake then contains the two pools:
+
+```mdtest-output
+coinflips@main
+coinflips@trial
+numbers@main
+```
+
+The following file `hello.zson` is also used.
+
+```mdtest-input hello.zson
+{greeting:"hello world!"}
+```
 
 ### Examples
 
-_Copy input to two paths and merge_
+_Source structured data from a local file_
+
+```mdtest-command
+zq -z 'file hello.zson | yield greeting'
+```
+=>
+```mdtest-output
+"hello world!"
+```
+
+_Source data from a local file, but in line format_
+```mdtest-command
+zq -z 'file hello.zson format line | head 1'
```

Comment:
Simplify this?
```suggestion
zq -z 'file hello.zson format line'
```

Reply:
@nwt: Derp. Yes, thank you. The `| head ` was a leftover from an earlier iteration where my ZSON file had multiple lines and I wanted to spare the reader excess output.